CardStackController/Classes/CardStackController.swift

```diff
@@ -133,6 +133,16 @@ public class CardStackController: UIViewController {
         imageView.pinEdgesToSuperviewEdges()
     }
 
+    public override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(false)
```

Comment:
Why `false`?

Reply:
Hm, I think I was just mimicking the approach for `viewWillAppear(_:)` without giving it too much thought 😅. Thinking we should just pass along the given `animated` value.